letsencrypt/data/run.sh

```diff
@@ -7,6 +7,17 @@ CERTFILE=$(bashio::config 'certfile')
 CHALLENGE=$(bashio::config 'challenge')
 DNS_PROVIDER=$(bashio::config 'dns.provider')
 
+if [[ "$CHALLENGE" == "dns" ]]; then
+  bashio::log.info "Selected DNS Provider: $(bashio::config 'dns.provider')"
+else
+  bashio::log.info "Selected http verification"
+fi
+
+if [[ "$EMAIL" == "hello@home-assistant.io" ]] || [[ "$DOMAINS" == "home-assistant.io" ]]; then
+  bashio::log.error "Please update the Email and Domain section with your details"
+  exit
+fi
```

Comment:
Is not needed with the comment above

Reply:
This was done because of the discussion in #881. Its correct that the error message, when using null is hard to understand.
Therefore using this approach would provide the user with an easy understandable error message, making the setup easier.